go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/git-lfs/go-netrc
+
+go 1.20
```

Comment:
I don't think we want to force users to use Go 1.20.  We'd probably prefer to specify the oldest version that works here, which is going to be something more like 1.13.